Automattic/wp-calypso
# desktop: Merge version 6.1.0

#### Changes proposed in this Pull Request

* Bumps the Desktop app version to 6.1.0-beta1.

#### Testing instructions

* Given this is just a version change, green CI should be enough. In fact, the whole reason to open this PR in the first place is to have the tests run in CI
